test/e2e/registration_service_test.go

```diff
@@ -429,6 +432,138 @@ func (s *registrationServiceTestSuite) TestSignupOK() {
 	assert.Equal(s.T(), ExpectedConsoleURL(s.T(), s.memberAwait, memberCluster), mp["consoleURL"])
 }
 
+func (s *registrationServiceTestSuite) TestPhoneVerification() {
+	// Create a token and identity to sign up with
+	identity0 := authsupport.NewIdentity()
+	emailValue := uuid.NewV4().String() + "@some.domain"
+	emailClaim0 := authsupport.WithEmailClaim(emailValue)
+	token0, err := authsupport.GenerateSignedE2ETestToken(*identity0, emailClaim0)
+	require.NoError(s.T(), err)
+
+	// Call the signup endpoint
+	invokeEndpoint(s.T(), "POST", s.route + "/api/v1/signup", token0, "", http.StatusAccepted)
```

Comment:
I have maybe a dumb question - couldn't we use the `postSignup` function for that - it looks like it's doing almost the same https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/user_setup.go#L124-L148
We could technically reuse the `invokeEndpoint` function there.

Reply:
I could use it, but it's not going to reduce the lines of code - the email claim and the token are still required as they are used further down in the test.